modules/ossm-add-project-using-label-selectors-console.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-mesh.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-label-selectors-console_{context}"]
+= Adding a project to the {SMProductShortName} using label selectors with the web console
+
+You can use labels selectors to add a project to the {SMProductShortName} with the {product-title} web console.
+
+.Prerequisites
+* An installed, verified {SMProductName} Operator.
+* An existing `ServiceMeshMemberRoll` resource.
+* To perform this task, a user must have mesh admin privileges.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the *Project* menu and choose the project where your `ServiceMeshMemberRoll` resource is deployed from the drop-down list. For example, `istio-system`.
```

Comment:
```suggestion
. Click the *Project* menu, and from the drop-down list, select the project where your `ServiceMeshMemberRoll` resource is deployed, such as *istio-system*.
```
- Per ISG: for drop-down lists, "se select to refer to selecting an item from a list." https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list
- Per ISG: use bold type for UI text and labels

Reply:
Prior peer reviews suggested breaking the "for example" part into a separate sentence, particularly when sentences are length. I'll keep the example as a separate clause, but thanks for the link about using select with drop-down menu. That's helpful for sure!

Done.